### ISSUE TITLE
Loss curve?

### ISSUE THREAD

**Issue by hkunzhe** [tianweiy/CausVid#8]
Thanks for the excellent project! Would you please share the loss curves for Bidirectional DMD Training, Causal ODE Pretraining and Causal DMD Training separately? Thank you in advance!

**Comment by tianweiy** [comment]
sorry, I just graduated and started new jobs and I do not have  access to my original log now ...

**Comment by hkunzhe** [comment]
@tianweiy Congrats on passing your PhD defense! My training loss curve is similar to yours (posted at https://wandb.ai/yin2/causvid/runs/8axhn7wn?nw=nwuseryin2). I found the generator loss of DMD2 on Wan2.1-1.3B didn't converge well, compared to DMD2 on SDXL (https://github.com/tianweiy/DMD2/issues/51). How to evaluate the quality (good/bad) of training for different runs?

**Comment by tianweiy** [comment]
I had similar observation and I don't have a good way to tell too unfortunately.. 

I just looked at sampled videos for some intuition.
